apps/compilersforhumans/src/pages/unsubscribed.tsx

```diff
@@ -20,14 +15,6 @@ const Unsubscribed = () => {
     }
   }
 
-  React.useEffect(() => {
-    if (tagId && !isEmpty(subscriber)) {
-      axios
-        .post('/api/convertkit/subscribe', {...subscriber, tag: tagId})
-        .catch((err) => console.log(err))
-    }
```

Comment:
this bit would be a "resubscribe" handler if we had that sort of button on the screen, but we probably don't need that.